x-pack/plugins/ml/common/constants/app.ts

```diff
@@ -38,3 +42,12 @@ export function initEnabledFeatures(enabledFeatures: MlFeatures, config: ConfigS
     enabledFeatures.nlp = config.nlp.enabled;
   }
 }
+
+export function initExperimentalFeatures(
+  experimentalFeatures: ExperimentalFeatures,
+  config: ConfigSchema
+) {
+  if (config.experimental?.ruleFormV2?.enabled !== undefined) {
```

Comment:
super small nitpick: maybe better to check if its a boolean?

Reply:
Trying to repeat the same pattern used elsewhere in this file